test/e2e/testdata/.snapshots/TestLinkToken/Create/POST__api_atlas_v2_orgs_a0123456789abcdef012345a_liveMigrations_linkTokens_1.snaphost

```diff
@@ -13,4 +13,4 @@ X-Java-Version: 17.0.14+7
 X-Mongodb-Service-Version: gitHash=b4fa165bfbcb725abc0816ef770e5f595c0528ce; versionString=master
 X-Permitted-Cross-Domain-Policies: none
 
-{"linkToken":"[REDACTED]"}
```

Comment:
You can do that or set the content length

Reply:
I think if you move your code to maskString it will automatically adjust content length

Reply:
cool didn't know about maskString, let me try

Reply:
updated!